common/envs/detailedEnvs.go

```diff
@@ -0,0 +1,10 @@
+package envs
+
+const NodeVersionConst = "NODE_VERSION"
+const PythonVersionConst = "PYTHON_VERSION"
+
+// EnvDetailsSeparatorMap is a map of environment variables and their separators
+var EnvDetailsSeparatorMap = map[string]string{
```

Comment:
should this be in `common`? I wonder if it makes more sense to place it in `procdiscovery`